packages/react-native-editor/android/app/build.gradle

```diff
@@ -135,17 +95,17 @@ dependencies {
     implementation "org.wordpress.gutenberg-mobile:react-native-bridge"
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation "com.google.android.material:material:1.9.0"
+    implementation "androidx.core:core-ktx:1.10.0"
     // The version of react-native is set by the React Native Gradle Plugin
     implementation "com.facebook.react:react-android"
-
-    implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
+    implementation("com.facebook.react:flipper-integration")
 
     implementation "com.github.wordpress-mobile:react-native-video:${extractPackageVersion(packageJson, 'react-native-video', 'dependencies')}"
     implementation "com.github.wordpress-mobile:react-native-slider:${extractPackageVersion(packageJson, '@react-native-community/slider', 'dependencies')}"
 
     // Published by `wordpress-mobile/react-native-libraries-publisher`
     // See the documentation for this value in `build.gradle.kts` of `wordpress-mobile/react-native-libraries-publisher`
-    def reactNativeLibrariesPublisherVersion = "v4"
+    def reactNativeLibrariesPublisherVersion = "v5"
```

Comment:
`v5` will be [released in this PR](https://github.com/wordpress-mobile/react-native-libraries-publisher/pull/33).